book/tutorials/example/intro-ipyleaflet.ipynb

```diff
@@ -5,15 +5,30 @@
    "id": "suffering-union",
    "metadata": {},
    "source": [
-    "# An introduction to rasters\n",
+    "# Example tutorial notebook\n",
     "\n",
-    "We will examine raster images from the [MODIS instrument](https://modis.gsfc.nasa.gov/data/). \"MODIS\" stands for \"MODerate Resolution SpectroRadiometer\". Moderate resolution refers to the fact that MODIS data has at best a 250 meter pixel posting, but single images cover hundreds of kilometers and with two sensors currently in orbit, we currently get [daily views of the entire globe](https://worldview.earthdata.nasa.gov/)!"
+    "```{admonition} Learning Objectives\n",
+    "**This is an example tutorial notebook for UW hackweeks**\n",
+    "- understand basic Jupyter Book formatting of .ipynb files\n",
+    "- use an open source Python library [ipyleaflet](https://github.com/jupyter-widgets/ipyleaflet) to create interactive maps\n",
+    "```\n",
+    "☝️ This formatting is a Jupyter Book [admonition](https://jupyterbook.org/content/content-blocks.html#notes-warnings-and-other-admonitions)"
+   ]
```

Comment:
Want to add another sentence and add the link to the multitude of options for Jupyter Books:
https://jupyterbook.org/content/index.html